debian/srptools.install

```diff
@@ -1,6 +1,11 @@
 etc/srp_daemon.conf
+lib/systemd/system/srp_daemon.service
+lib/systemd/system/srp_daemon_port@.service
 usr/sbin/ibsrpdm
 usr/sbin/srp_daemon
+libexec/srp_daemon/start_on_all_ports
 usr/share/doc/rdma-core/ibsrpdm.md usr/share/doc/srptools/
```

Comment:
The libexec dir is usr/lib/ on Debian - looks like this is why travis is failing?